symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/MessageBuilder.java

```diff
@@ -121,25 +117,4 @@ public MessageBuilder attachment(InputStream inputStream, String filename) {
   public Message build() {
     return new Message(this);
```

Comment:
maybe we should provider some sort of validation to ensure that we build a valid message

Reply:
Validation was done in `Message` constructor, now moved in `build()` method.